mongodb/src/test/scala/docs/scaladsl/MongoSinkSpec.scala

```diff
@@ -42,7 +43,7 @@ class MongoSinkSpec
   implicit val system = ActorSystem()
 
   override protected def beforeAll(): Unit =
-    Source.fromPublisher(db.drop()).runWith(Sink.headOption).futureValue
+    Source.fromPublisher(db.drop()).runWith(Sink.headOption).futureValue(defaultPatience, Position.here)
```

Comment:
Not sure why, but this failed 5 times in a row in CI, so...

Reply:
Hmm, isn't defaultPatience already what futureValue picks up implicitly?

Reply:
It is, and I dont see why my changes could impact this. However, that place as consistently failing for a timeout until I added this: https://github.com/akka/alpakka/actions/runs/4253962480/jobs/7400425654.

Should I remove this explicit params again, and see what happens? This was also only showing up in CI.

Reply:
Maybe instead increase the default patience for this test up top and keep the implicit timeout pickup as it was?

Something like
```
override implicit val patienceConfig: PatienceConfig = PatienceConfig(10.seconds, 100.millis)
```

Reply:
Sorry, looks like the name to override here is `defaultPatience`

Reply:
👍